robolectric/src/test/java/org/robolectric/shadows/ShadowDisplayManagerTest.java

```diff
@@ -513,6 +518,87 @@ public void configureDefaultDisplay_calledTwice_showsReasonableException() {
         .contains("configureDefaultDisplay was called a second time");
   }
 
+  @Test
+  @Config(minSdk = Build.VERSION_CODES.N)
+  public void displayManager_activityContext_retrievesDisplaysWithMultiWindow() {
+    String originalProperty = System.getProperty("robolectric.createActivityContexts", "");
+    System.setProperty("robolectric.createActivityContexts", "true");
+    Activity activity = null;
+    try {
+      activity = Robolectric.setupActivity(TestActivity.class);
+      DisplayManager applicationDisplayManager =
+          (DisplayManager)
+              ApplicationProvider.getApplicationContext().getSystemService(Context.DISPLAY_SERVICE);
+      DisplayManager activityDisplayManager =
+          (DisplayManager) activity.getSystemService(Context.DISPLAY_SERVICE);
+
+      Display[] applicationDisplays = applicationDisplayManager.getDisplays();
+      Display[] activityDisplays = activityDisplayManager.getDisplays();
+
+      assertThat(applicationDisplays).isNotEmpty();
+      assertThat(activityDisplays).isNotEmpty();
+      assertThat(activityDisplays.length).isEqualTo(applicationDisplays.length);
+
+      for (int i = 0; i < applicationDisplays.length; i++) {
+        Display appDisplay = applicationDisplays[i];
+        Display actDisplay = activityDisplays[i];
+
+        assertThat(actDisplay.getDisplayId()).isEqualTo(appDisplay.getDisplayId());
+        assertThat(actDisplay.getWidth()).isEqualTo(appDisplay.getWidth());
+        assertThat(actDisplay.getHeight()).isEqualTo(appDisplay.getHeight());
+      }
+
+      Display originalDisplay = applicationDisplays[0];
+      int originalWidth = originalDisplay.getWidth();
+      int originalHeight = originalDisplay.getHeight();
+
+      setMultiWindowMode(activity, true);
+
+      Display[] multiWindowDisplays = activityDisplayManager.getDisplays();
+      assertThat(multiWindowDisplays).isNotEmpty();
+
+      for (Display mwDisplay : multiWindowDisplays) {
+        int mwWidth = mwDisplay.getWidth();
+        int mwHeight = mwDisplay.getHeight();
+
+        assertThat(mwWidth).isAtMost(originalWidth);
```

Comment:
> I think they're not same, and it's also correct if we run it on Emulator as the display size accessed by application is real screen size, and the display size accessed by activity is window's size. The window's size is equals to screen size if window is full screen. But it will be different when the window is in the multi-window mode,including freeform, pip and split screen.

I have tried this assertion 
"assertThat(mwWidth).isLessThan(originalWidth);
        assertThat(mwHeight).isLessThan(originalHeight);"
        
   but it is failing and little confused about that , can you please guide me how can I fix this